app/assets/stylesheets/config/_fonts.scss

```diff
@@ -1,13 +1,54 @@
-// Define fonts for body and headers
-$body-font: "Helvetica", "sans-serif";
-$headers-font: "Helvetica", "sans-serif";
+// Generated with https://google-webfonts-helper.herokuapp.com/fonts/poppins?subsets=latin
 
-// To use a font file (.woff) uncomment following lines
-// @font-face {
-//   font-family: "Font Name";
-//   src: font-url('FontFile.eot');
-//   src: font-url('FontFile.eot?#iefix') format('embedded-opentype'),
-//        font-url('FontFile.woff') format('woff'),
-//        font-url('FontFile.ttf') format('truetype')
-// }
-// $my-font: "Font Name";
+/* poppins-regular - latin */
+@font-face {
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 400;
+  src: url("../fonts/poppins-v15-latin-regular.eot"); /* IE9 Compat Modes */
+  src: local(""),
+    url("../fonts/poppins-v15-latin-regular.eot?#iefix")
+      format("embedded-opentype"),
+    /* IE6-IE8 */ url("../fonts/poppins-v15-latin-regular.woff2")
+      format("woff2"),
+    /* Super Modern Browsers */ url("../fonts/poppins-v15-latin-regular.woff")
+      format("woff"),
+    /* Modern Browsers */ url("../fonts/poppins-v15-latin-regular.ttf")
+      format("truetype"),
+    /* Safari, Android, iOS */
+      url("../fonts/poppins-v15-latin-regular.svg#Poppins") format("svg"); /* Legacy iOS */
+}
+
+/* poppins-500 - latin */
+@font-face {
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 500;
+  src: url("../fonts/poppins-v15-latin-500.eot"); /* IE9 Compat Modes */
+  src: local(""),
+    url("../fonts/poppins-v15-latin-500.eot?#iefix") format("embedded-opentype"),
```

Comment:
Normalement en Rails ont mets les fonts dans `app/assets/fonts` et on utilise [`asset-url(non_du_ficher)`](https://guides.rubyonrails.org/asset_pipeline.html#css-and-sass) pour aller les chercher. Ça permet de bénéficier de tous les avantages de l' [Asset Pipleine](https://guides.rubyonrails.org/asset_pipeline.html)

Reply:
Ah je me posais la question, je les y avait mis puis remis dans public car je me suis dit qu'elles n'avaient pas besoin de préprocessing. Mais en y regardant de plus près y'a le gzipping et tout.

Je les y déplace 🙏